pkg/bundle/multi_scoped_cache/cache.go

```diff
@@ -51,23 +56,23 @@ type multiScopedCache struct {
 // resources in the given namespace. namespacedInformers is the set of resource
 // types which should only be watched in the given namespace.
 // namespacedInformers expects Namespaced resource types.
-func NewMultiScopedCache(namespace string, namespacedInformers []schema.GroupKind) cache.NewCacheFunc {
+func NewMultiScopedCache(scheme *runtime.Scheme, namespace string, namespacedInformers []schema.GroupKind) cache.NewCacheFunc {
```

Comment:
Why do we need to pass a scheme? What's wrong with the cache Options scheme?

Reply:
@JoshVanL that pointer can be nil, how would you get a default value in case that pointer is nil?

Reply:
If nil is not acceptable, we can default to runtime.NewScheme()